docs/versioned_docs/version-4.1.x/releases.md

```diff
@@ -58,12 +72,36 @@ Grafana Dashboards for Replicated PV Mayastor has been added in this releases.
 
 ### Fixed Issues - Local Storage
 
+- **Environment Variable**
+
+Ensures that the environment variable to disable event analytics reporting is properly honored. ([#211](https://github.com/openebs/dynamic-localpv-provisioner/pull/211))
+
+- **Prevent Race Condition**
+
+Adds `priorityClassName` to pods to prevent race conditions caused by pod eviction. ([#182](https://github.com/openebs/dynamic-localpv-provisioner/pull/182))
+
 - **Metrics Collection Loop**
 
 Adds an anonymous metrics collection loop which periodically pushes OpenEBS usage metrics. ([#188](https://github.com/openebs/dynamic-localpv-provisioner/pull/188),[#318](https://github.com/openebs/lvm-localpv/pull/318), and[#548](https://github.com/openebs/zfs-localpv/pull/548))
 
 ### Fixed Issues - Replicated Storage
 
+- **Make Readiness Check Atomic in REST API**
+
+Ensures the readiness check API performs atomic operations to prevent multiple concurrent probes of the agent-core service.
+
+- **Prevent volume Unpublish from Non-frontend Nodes**
```

Comment:
What do we mean by frontend and non-frontend nodes?

Reply:
Nodes where apps are deployed --> frontend nodes.
Other nodes, nodes which may house replicas, nexuses, host rebuilds, but strictly don't contain any application pods which are consuming volumes.

cc: @Abhinandan-Purkait @tiagolobocastro